pilot/pkg/config/kube/gateway/backend_policies.go

```diff
@@ -0,0 +1,423 @@
+	return krt.NewStatusManyCollection(tlsPolicies, func(ctx krt.HandlerContext, i *gatewayalpha3.BackendTLSPolicy) (
+		*gatewayalpha2.PolicyStatus,
+		[]BackendPolicy,
+	) {
+		status := i.Status.DeepCopy()
+		res := make([]BackendPolicy, 0, len(i.Spec.TargetRefs))
+		ancestors := make([]gatewayalpha2.PolicyAncestorStatus, 0, len(i.Spec.TargetRefs))
+
+		tls := &networking.ClientTLSSettings{Mode: networking.ClientTLSSettings_SIMPLE}
+		s := i.Spec
+
+		conds := map[string]*condition{
+			string(gatewayalpha2.PolicyConditionAccepted): {
+				reason:  string(gatewayalpha2.PolicyReasonAccepted),
+				message: "Configuration is valid",
+			},
+		}
+		tls.Sni = string(s.Validation.Hostname)
+		tls.SubjectAltNames = slices.MapFilter(s.Validation.SubjectAltNames, func(e gatewayalpha3.SubjectAltName) *string {
+			switch e.Type {
+			case gatewayalpha3.HostnameSubjectAltNameType:
+				return ptr.Of(string(e.Hostname))
+			case gatewayalpha3.URISubjectAltNameType:
+				return ptr.Of(string(e.URI))
+			}
+			return nil
+		})
+		if wk := s.Validation.WellKnownCACertificates; wk != nil {
+			switch *wk {
+			case gatewayalpha3.WellKnownCACertificatesSystem:
+				// Already our default, no action needed
+			default:
+				conds[string(gatewayalpha2.PolicyConditionAccepted)].error = &ConfigError{
+					Reason:  string(gatewayalpha2.PolicyReasonInvalid),
+					Message: fmt.Sprintf("Unknown wellKnownCACertificates: %v", *wk),
+				}
+			}
+		} else if len(s.Validation.CACertificateRefs) > 0 {
+			// Spec should require but double check
+			// We only support 1
+			ref := s.Validation.CACertificateRefs[0]
+			refo, err := references.LocalPolicyRef(ref, i.Namespace)
+			if err == nil {
+				switch to := refo.(type) {
+				case *v1.ConfigMap:
+					if _, rerr := kubesecrets.ExtractRootFromString(to.Data); rerr != nil {
+						err = rerr
+					}
+					tls.CredentialName = credentials.KubernetesConfigMapTypeURI + i.Namespace + "/" + string(ref.Name)
+				// TODO: for now we do not support Secret references.
+				// Core requires only ConfigMap
+				// We can do so, we just need to make it so this propagates through to SecretAllowed, otherwise clients in other namespaces
+				// will not be given access.
+				// Additionally, we will need to ensure we don't accidentally authorize them to access the private key, just the ca.crt
+				default:
+					err = fmt.Errorf("unsupported reference kind: %v", ref.Kind)
+				}
+			}
+			if err != nil {
+				conds[string(gatewayalpha2.PolicyConditionAccepted)].error = &ConfigError{
+					Reason:  string(gatewayalpha2.PolicyReasonInvalid),
+					Message: fmt.Sprintf("Certificate reference invalid: %v", err),
+				}
+				// Generate an invalid reference. This ensures traffic is blocked.
+				// See https://github.com/kubernetes-sigs/gateway-api/issues/3516 for upstream clarification on desired behavior here.
+				tls.CredentialName = "invalid://"
+			} else if len(s.Validation.CACertificateRefs) > 1 {
```

Comment:
nit: feels like this should be first based on the comment

Reply:
I am not sure what you mean

Reply:
Sorry, poorly worded. I think this else if should be higher up so it's close to the comment about only allowing 1. Not sure if you organized it this way to try and avoid cyclomatic complexity